plane/source/docs/fixed-wing-faq.rst

```diff
@@ -177,7 +177,7 @@ slow an aircraft for landing without inducing a lot of pitching moment.
 
 To setup crow flaps you will need to combine two features. First you
 will need to :ref:`setup flaperons <flaperons-on-plane>` on two
-output channels using the :ref:`flaperon output channels functions <channel-output-functions_flaperon1_and_flaperon2>`.
+output channels using the Right.
```

Comment:
This English seems a bit strange though.. "using the Right".. is a word missing?

Reply:
very strange...its not what I had tested and thought committed...
"output channels using the FlaperonLeft and FlaperonRight functions"
will re-commit...

Reply:
PS this was to fix just build error, but I had to do broader work later on flaperons to correct other errors, not build error...PR #1889

Reply:
no better to just fix this one since there are so many small changes...should be fine now...must have had two windows open...last change was to capitalize the R in FlaperonRight....did not test  it...that will teach me to check even one letter changes